amp/system-oracle/Dockerfile

```diff
@@ -7,6 +7,9 @@ COPY ./oracle-client-files/instantclient-basic*-linux.x64-12.2.*.zip \
      ./oracle-client-files/instantclient-odbc-linux.x64-12.2.*.zip \
      /opt/oracle/
 
+COPY ./oracle-client-files/vendor/cache/activerecord-oracle_enhanced-adapter-1.6.9.gem /opt/system/vendor/cache/
+COPY Gemfile.oracle Gemfile.oracle.lock /opt/system/
```

Comment:
No need of this Gemfiles if the `activerecord-oracle_enhanced-adapter-1.6.9.gem` and dependencies are already present.